bin/compile

```diff
@@ -274,6 +276,16 @@ sub_env "$BIN_DIR/steps/geo-libs"
 # shellcheck source=bin/steps/gdal
 source "$BIN_DIR/steps/gdal"
 
+# SQLite3 support.
+# This sets up and installs sqlite3 dev headers and the sqlite3 binary but not the
+# libsqlite3-0 library since that exists on the stack image.
+# Note: This only applies to Python 2.7.15+ and Python 3.6.5+
+(( start=$(nowms) ))
+# shellcheck source=bin/steps/sqlite3
+source "$BIN_DIR/steps/sqlite3"
```

Comment:
We need to use it in both since the dev headers are missing on the stack image, so it won't be around for build or runtime. We need it at build time, so the python sqlite library is built against the version of sqlite3 on the stack image.